config/gitconfig.dotfiles

```diff
@@ -4,8 +4,14 @@
 	autoSquash = true
 [alias]
 	br = branch
+	ci = commit
 	co = checkout
+	cof = !git checkout master && git fetch && git checkout $1 && :
+	cob = !git checkout master && git fetch && git checkout -b $1 && :
 	cp = cherry-pick
 	m = !sh -c 'exec git merge --no-ff origin/$0'
-	unpushed = log --branches --not --remotes
 	sha1 = rev-parse HEAD
+	st = status
+	unpushed = log --branches --not --remotes
+[pull]
+	rebase = merges
```

Comment:
This value requires git >= 2.22. Not entirely sure what the difference between "merges" and "preserve" (which is also available in older versions) is, but since I still run Debian stretch (git 2.11) in a number of places, including work, I'm sceptical to include this. I'm happy to include the other lines.

Reply:
Installing git 2.20 from stretch-backports solves this.